distr_tracing/distr_tracing_tempo/distr-tracing-tempo-removing.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: ASSEMBLY
+[id="dist-tracing-tempo-removing"]
+= Removing distributed tracing platform Tempo
+include::_attributes/common-attributes.adoc[]
+:context: dist-tracing-tempo-removing
+
+toc::[]
+
+The steps for removing {DTProductName} from an {product-title} cluster are as follows:
+
+. Shut down any {DTProductName} pods.
+. Remove any {DTProductName} instances.
+. Remove the {TempoName} Operator.
+. Remove the {OTELName} Operator.
```

Comment:
if the otel operator was installed. It something we recommend together, but it doesnt depend on each other.